static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -142,8 +149,17 @@ function CodecovLink({codecovUrl, codecovStatusCode}: CodecovLinkProps) {
     if (!codecovUrl) {
       return null;
     }
+
+    const onOpenCodecovLink = () => {
+      trackIntegrationAnalytics('integrations.stacktrace_codecov_link_clicked', {
```

Comment:
```suggestion
      trackIntegrationAnalytics(StacktraceLinkEvents.CODECOV_LINK_CLICKED, {
```

Reply:
The rest of the `trackIntegrationAnalytics` function calls use `trackIntegrationAnalytics('integrations.<name>`. Should I change these too to make it more consistent?

Reply:
that would be great!